conjurapi/logging/logging.go

```diff
@@ -0,0 +1,35 @@
+package logging
+
+import (
+	"io"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+var ApiLog = logrus.New()
+
+func init() {
+	dest, ok := os.LookupEnv("CONJURAPI_LOG")
+	if !ok {
+		return
+	}
+
+	var out io.Writer
+	switch dest {
+	case "stdout":
+		out = os.Stdout
+	case "stderr":
+		out = os.Stderr
+	default:
+		f, err := os.OpenFile(dest, os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			logrus.Fatalf("Failed to %s", err.Error())
+			return
+		}
+		out = f
```

Comment:
Why not assign `out` when you're opening the file? If there's an error, the program should exit I think.